AudioCompression/src/audioCompression/algorithm/FilterBankStep.java

```diff
@@ -82,6 +82,8 @@ public RawAudio reverse(Subbands input) {
 		}
 		WavAudioOutput out = 
 				new WavAudioOutput(windows, input.getWindowOverlap(), input.getSampleRate());
+		
+		out.writeFile(name);
```

Comment:
Perhaps it would be best to have another stage at the beginning of the pipeline that explicitly loads the input file, and writes the decompressed wav file (forward / reverse).  Then it is very clear what each stage is doing, and others could be added in between if necessary.